roles/pulp_health_check/tasks/main.yml

```diff
@@ -0,0 +1,39 @@
+---
+- meta: flush_handlers
+
+- name: Wait 5 seconds
+  pause:
+    seconds: "5"
```

Comment:
Please provide a comment for why we do this. Sleeps are universally considered bad form in programming.

Reply:
I believe we don't need it anymore, I'll remove